cli/src/bin/phylum.rs

```diff
@@ -147,7 +147,7 @@ async fn handle_commands() -> CommandResult {
         #[cfg(feature = "selfmanage")]
         "uninstall" => uninstall::handle_uninstall(sub_matches),
 
-        "extension" => extensions::handle_extensions(Box::pin(api), sub_matches).await,
+        "extension" => extensions::handle_extensions(app_helper, Box::pin(api), sub_matches).await,
```

Comment:
Not sure but we might want to keep API as first parameter just for consistency with everything else?

Reply:
Makes sense to me -- I put it as last argument to also preserve consistency with other methods having `sub_matches` as second argument.